src/metrics.jl

```diff
@@ -86,7 +86,7 @@ julia> pairwise(Euclidean(1e-12), x, x)
  0.0
 ```
 """
-Euclidean() = Euclidean(0)
+    Euclidean() = Euclidean(0)
```

Comment:
OH WOW I'M SO DUMB. I thought this was the top of the docstring when I was doing my initial review so I told @richardreeve to indent it. Sorry!